extendr-api/Cargo.toml

```diff
@@ -14,6 +14,7 @@ edition = "2021"
 description = "Safe and user friendly bindings to the R programming language."
 license = "MIT"
 repository = "https://github.com/extendr/extendr"
+rust-version = "1.60"
```

Comment:
The other crates are dependencies of this crate and therefore unnecessary.

Reply:
Hmm, I wonder if there's any reason not to add it anyway. Not sure either way.

Reply:
No harm in adding it, it just seemed redundant to me to write the same thing in multiple places.

Rust 1.64 supports inheritance in the workspace, so it can be set for all crates by specifying the version only in the root Cargo.toml.
However, since we are assuming Rust 1.60 here, workspace inheritance cannot be used.